src/test/java/Homework17.java

```diff
@@ -0,0 +1,60 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        String expectedSongAddedMessege = "Added 1 song into \"Love.\" ";
```

Comment:
Hello, your homework 17 has been reviewed again but the test script execute because of a small error: a extra space at expectedSongAddedMessege. Please fix it
![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/144403954/7611fb84-0099-4518-affc-e14e6c188cb0)


//HINT: `String expectedSongAddedMessege = "Added 1 song into \"Love.\"";`